examples/raylib/build.zig

```diff
@@ -0,0 +1,86 @@
+const android = @import("android");
+const std = @import("std");
+
+//this is targeting android API level 29. 
+//You may have to change the values here and in android/AndroidManifest.xml to target your desired API level
+const android_api = "29";
+const android_version = .android10;
```

Comment:
This will keep both values synchronized.
```zig
const android_api = std.fmt.comptimePrint("{}", .{@intFromEnum(android_version)});
const android_version: android.APILevel = .android10;
```